src/picongpu/include/simulation_defines/unitless/speciesAttributes.unitless

```diff
@@ -109,7 +109,7 @@ struct Unit<globalCellIdx<T_Type> >
 };
 
 template<>
-struct Unit<chargeState >
+struct Unit<boundElectrons>
 {
   static std::vector<double> get()
   {
```

Comment:
I guess that should be done for all of them in one PR. That would be far more effective than doing each one separately.